app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -241,7 +241,11 @@ public void onBackPressed() {
             contributionsFragment.backButtonClicked();
         } else if (nearbyParentFragment != null && activeFragment == ActiveFragment.NEARBY) {
             // Means that nearby fragment is visible
-            nearbyParentFragment.backButtonClicked();
+           //  function nearbyParentFragment.backButtonClick() return false mean bottomsheet is not expand then if backbutton pressed then come to Contributions tab
```

Comment:
Minor change: Could you please rephrase to: `If function nearbyParentFragment.backButtonClick() returns false, it means that the bottomsheet is not expanded. So if the back button is pressed, then go back to the Contributions tab`. Thanks!